eng/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -1,30 +1,53 @@
 parameters:
   LiveTestSamples: []
 
-
 jobs:
-  - job: LiveTest
+  - job:
+    displayName: LiveTest
     condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
     strategy:
       matrix:
         Linux_x64_with_iot_samples:
-          Pool: azsdk-pool-mms-ubuntu-2004-general
-          OSVmImage: ubuntu-20.04
+          Pool: $(LINUXPOOL)
+          OSVmImage: $(LINUXVMIMAGE)
           VCPKG_DEFAULT_TRIPLET: 'x64-linux'
           build.args: ' -DTRANSPORT_PAHO=ON'
           test_type: 'iot'
           os: 'linux'
         Linux_ubuntu_2204_x64_with_iot_samples:
-          Pool: azsdk-pool-mms-ubuntu-2204-general
-          OSVmImage: ubuntu-22.04
+          Pool: azsdk-pool-mms-ubuntu-2204-1es-pt
+          OSVmImage: azsdk-pool-mms-ubuntu-2204-1espt
           VCPKG_DEFAULT_TRIPLET: 'x64-linux'
           build.args: ' -DTRANSPORT_PAHO=ON'
           test_type: 'iot'
           os: 'linux'
+        # Disable live testing for these matrix entries until we have something to
```

Comment:
Just keeping the commented builds around [from main](https://github.com/Azure/azure-sdk-for-c/blob/2dce952c305249fe24f848d09e5acb957240af9e/eng/pipelines/templates/jobs/archetype-sdk-tests.yml#L51)